Rebranding UI Comparison Table

hasparus opened the pull request:
## Description

Added a Comparison Table for the landing page.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Screenshots/Sandbox (if appropriate/relevant):

<img width="1477" alt="image" src="https://github.com/user-attachments/assets/146d34b2-1169-4a0d-b8b7-139f9bb8bcd6" />
<img width="619" alt="image" src="https://github.com/user-attachments/assets/cec8d104-16b8-4790-8387-8073b44abafa" />

hasparus commented on website/src/components/index-page.tsx:
```diff
@@ -6,6 +6,7 @@ import meshDiagram from '@/public/assets/mesh-diagram.svg';
 import meshExampleLogo from '@/public/assets/mesh-example.png';
 import openSourceLogo from '@/public/assets/open-source.svg';
 import { Anchor } from '@theguild/components';
+import { ComparisonTable } from './comparison-table';
```
```suggestion
```

hasparus commented on website/src/components/index-page.tsx:
```diff
@@ -209,6 +210,7 @@ export function IndexPage(): ReactElement {
       <div className={FeatureWrapperClass}>
         <Hero />
       </div>
+      <ComparisonTable />
```
```suggestion
```

dimaMachina commented on website/src/components/comparison-table/index.tsx:
```diff
@@ -0,0 +1,165 @@
+      width="16"
+      height="17"
+      viewBox="0 0 16 17"
+      className={cn('text-green-800', className)}
+      fill={filled === 2 ? 'currentColor' : 'none'}
```
I think there is a way to simplify it? just by passing `className` as props?

ardatan commented:
@hasparus I think some changesets are pushed by accident. Could you revert them? Thanks!

hasparus commented:
> chore(dependencies): updated changesets for modified dependencies

I wanted to ask, why does the GitHub Action add them when I didn't modify package.json nor the lockfile?

[chore(dependencies): updated changesets for modified dependencies](https://github.com/ardatan/graphql-mesh/pull/8178/commits/f01d4c3755cabf3e5c9bfbe70573e6d650f4adcb)
